code/player/TTTPlayer.AFK.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Sandbox;
+
+using TTTReborn.Settings;
+
+namespace TTTReborn.Player
+{
+    public partial class TTTPlayer
+    {
+        public readonly static List<InputButton> Buttons = Enum.GetValues(typeof(InputButton)).Cast<InputButton>().ToList();
+        private static int SecondsTillKick => ServerSettings.Instance.AFK.MinutesTillKick * 60;
+
+        private TimeSince _timeSinceLastAction = 0f;
+        private Rotation? _lastKnownRotation;
```

Comment:
Do we really need to track rotation as well? I think that tracking button presses are enough, or even just tracking rotation could work. Open to hearing thoughts.

Reply:
This is for instances where the player isn't moving but is looking around.

It's not super big to do, or not do. I just figured it was another small value I could check against to see if it was different. Open for a different value, but I think the check should be more than just a button check.

Reply:
Removed, and replaed with a check `Input.MouseDelta != Vector3.Zero`